specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ManagedEnvironments.json

```diff
@@ -308,6 +308,326 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/managedEnvironments/{managedEnvironmentName}/certificates": {
+      "get": {
+        "tags": [
+          "ManagedEnvironments"
+        ],
+        "summary": "Get the Certificates in a given managed environment.",
+        "operationId": "Certificates_ListByManagedEnvironment",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "managedEnvironmentName",
+            "in": "path",
+            "description": "Name of the Managed Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CertificateCollection"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Certificates by Managed Environment": {
+            "$ref": "./examples/Certificates_ListByManagedEnvironment.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      },
+      "put": {
```

Comment:
Are we going to have an API that allows to PUT the entire collection? We did not have it for any other APIs and I wonder if it is needed. Maybe let's not add it yet until we know for sure it is needed.

Reply:
If we don't allow it, then it means certificates can only be PUT one-by-one. Happy to remove it if we are ok with it.